docker-compose.yaml

```diff
@@ -258,6 +258,13 @@ services:
       <<: *airflow-common-depends-on
       airflow-init:
         condition: service_completed_successfully
-# postgres db data persisted here
 volumes:
+  # postgres db data persisted here
   postgres-db-volume:
+  # volume for Knack SSL certs which will be mounted to DockerOperator tasks for 311 integration
+  knack-certs:
+    driver: local
+    driver_opts:
+      type: 'none'
+      o: 'bind'
+      device: ${AIRFLOW_PROJ_DIR}/certs/knack
```

Comment:
we are loading our local copy of the knack ssl certificate directory into the airflow stack as a named volume, which can then be referenced in the `DockerOperator` constructor. 

i tried a few alternatives, such as bind-mounting this directory in the `volumes` section of the `airflow-common-env`, but this is the only way i could get this to work.

Reply:
another option is to create a new env var called `KNACK_CERT_DIR` so that each we could have a more standard path to them, e.g.  `/etc/ssl/certs`.

Reply:
I'm a fan of the way you've done it. I think as much as we can, packing everything airflow related into something inside the repo is the way to go. If we wanted to more tightly integrate with the OS and its environment, we would probably be best served just running the airflow services as non-containerized processes on the machine, and that's just a big departure from everything we've done so far.